### PR TITLE
feat(inputs.netflow): Add sFlow decoder

### PR DESCRIPTION
**Pull request by srebhan**
- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

This PR adds a sflow decoder to the netflow plugin.

### CODE REVIEW

**Review by Hipska — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -84,6 +84,7 @@ require (
 	github.com/google/gnxi v0.0.0-20221016143401-2aeceb5a2901
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-github/v32 v32.1.0
+	github.com/google/gopacket v1.1.19
```
[the remark this answers is not in the document]
Comment (srebhan) [reply go.mod:87]: @powersj I do not see other libraries with the same functionality. The only alternative would be to manually decode the binary network frames which is well beyond the scope. The library is maintained (last commit on  Aug 10, 2022)  even though the commit rate is not high...

**Comment by powersj** [comment]
ok I think then we need tests cleaned up and some feedback from the user.

**Review by powersj — APPROVED** [review]

**Comment by powersj** [comment]
@srebhan I'm going to assign this to you and let you merge when you are ready. Not sure if you still want to wait for additional feedback from users.

**Comment by srebhan** [comment]
Let's merge it and adapt it to the feedback once it comes...

**Pull request merged**